### ISSUE TITLE
FEAT: Allow restoring backups to dockerized databases.

### ISSUE THREAD

**Issue by Futuremappermydud** [beekeeper-studio/beekeeper-studio#1885]
**Is your feature request related to a problem? Please describe.**

I want to use database backups that I've generated but I can't restore them with the tool.

**Describe the solution you'd like**

I would like if the tool could connect to the docker container and execute the commands.

**Describe alternatives you've considered**

**Additional context**
Add any other context or screenshots about the feature request here. (we love screenshots with red circles if the request is for something visual)

**Database type, if appropriate**
Does this feature only relate to a specific type of database? (eg Postgresql)

PostgreSQL in docker

**Comment by not-night-but** [comment]
Hey @Futuremappermydud ! What behaviour are you seeing with the docker restore? I developed this feature with my database in docker, and I just now ran a backup/restore with a postgres docker container. It does require that you have pg_dump/pg_restore on the host machine though. Is that something you're wanting to avoid?

**Comment by Futuremappermydud** [comment]
> Hey @Futuremappermydud ! What behaviour are you seeing with the docker restore? I developed this feature with my database in docker, and I just now ran a backup/restore with a postgres docker container. It does require that you have pg_dump/pg_restore on the host machine though. Is that something you're wanting to avoid?
> 
> Thanks so much for using Beekeeper and filing this issue! Day

Yes i was hoping it could just run docker exec with all the commands without needing it on my host machine. Didn’t know it’d still work so I’ll try it out with it installed
